build/optaplanner-ide-config/ide-configuration.adoc

```diff
@@ -18,4 +18,21 @@ Restart your IDE, open the _Preferences_ (or _Settings_) window again and naviga
 
 Select _Use the Eclipse Code Formatter_, then change the _Eclipse Java Formatter Config File_ to point to the `eclipse-format.xml`
 file in the `build/optaplanner-ide-config/src/main/resources/` directory.
-Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
\ No newline at end of file
+Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
+
+== VS Code setup
+. Open the _Extentions_ window (Ctrl+Shift+X), search for _Language Support for Java(TM) by Red Hat_ and install it.
+. Add import order to settings.json:
```

Comment:
Where do I find the `settings.json`?

Reply:
It is a file that is automatically gets opened when you click _bottom right gear_ -> Settings or use hot key Ctrl+Comma
That is the common way to setup VS code. :bulb: should I add that?

Reply:
Added that it should be in the root of your folder in case of workspace settings
You can check it by following the steps and to be sure that no default formating get used you can set the property 

<setting id="org.eclipse.jdt.core.formatter.blank_lines_after_imports" value="10"/>
And press Ctrl+Shift+I

Arranging imports: 
Shift+Alt+O

Reply:
@rsynek feel free to try it